spec/abci/abci.md

```diff
@@ -381,6 +402,83 @@ Commit are included in the header of the next block.
     function of anything that did not come from the
     BeginBlock/DeliverTx/EndBlock methods.
 
+### ListSnapshots
+
+- **Response**:
+  - `Snapshots ([]Snapshot)`: List of local state snapshots.
+- **Usage**:
+  - Used during state sync to discover available snapshots on peers.
+  - See `Snapshot` data type for details.
+
+### LoadSnapshotChunk
+
+- **Request**:
+  - `Height (uint64)`: The height of the snapshot the chunks belongs to.
+  - `Format (uint32)`: The application-specific format of the snapshot the chunk belongs to.
+  - `Chunk (uint32)`: The chunk index, starting from `0` for the initial chunk.
+- **Response**:
+  - `Chunk ([]byte)`: The binary chunk contents, in an arbitray format. Chunk messages cannot be
+    larger than 16 MB _including metadata_, so 10 MB is a good starting point.
+- **Usage**:
+  - Used during state sync to retrieve snapshot chunks from peers.
+
+### OfferSnapshot
+
+- **Request**:
+  - `Snapshot (Snapshot)`: The snapshot offered for restoration.
+  - `AppHash ([]byte)`: The light client-verified app hash for this height, from the blockchain.
+- **Response**:
+  - `Result (Result)`: The result of the snapshot offer.
+    - `accept`: Snapshot is accepted, start applying chunks.
+    - `abort`: Abort all snapshot restoration, including other snapshots.
+    - `reject`: Reject this specific snapshot, try others.
+    - `reject_format`: Reject all snapshots with this `format`, try others.
+    - `reject_senders`: Reject all snapshots from all senders of this snapshot, try others.
+- **Usage**:
+  - `OfferSnapshot` is called when bootstrapping a node using state sync. The application may
```

Comment:
How can we return `reject|abort` in the response to `OfferSnapshot` once we start accepting and receiving chunks via `ApplySnapshotChunk`?